### ISSUE TITLE
Guava 19 and GWTQuery: Referencing method 'com.google.gwt.query.client.GQuery.delegate'

### ISSUE THREAD

**Issue by confile** [ArcBees/gwtquery#370]
Using Guava 19 und GWTQuery leads to this warning when running in SDM: 

```
    Adding '1' new generated units
            [WARN] Warnings in '/Users/mg/Documents/Grails/GGTS3.6.4-GWT/majestella-client-chat/build/putnami/work/majestella.widget.MajestellaClientChat/compile-2/gen/com/google/gwt/query/client/LazyGQueryImpl.java'
               [WARN] Line 539: Referencing method 'com.google.gwt.query.client.GQuery.delegate': method 'com.google.gwt.query.client.GQuery.delegate(Ljava/lang/String;I[Lcom/google/gwt/query/client/Function;)' is deprecated
               [WARN] Line 548: Referencing method 'com.google.gwt.query.client.GQuery.delegate': method 'com.google.gwt.query.client.GQuery.delegate(Ljava/lang/String;ILjava/lang/Object;[Lcom/google/gwt/query/client/Function;)' is deprecated
               [WARN] Line 557: Referencing method 'com.google.gwt.query.client.GQuery.delegate': method 'com.google.gwt.query.client.GQuery.delegate(Ljava/lang/String;Ljava/lang/String;[Lcom/google/gwt/query/client/Function;)' is deprecated
               [WARN] Line 566: Referencing method 'com.google.gwt.query.client.GQuery.delegate': method 'com.google.gwt.query.client.GQuery.delegate(Ljava/lang/String;Ljava/lang/String;Ljava/lang/Object;[Lcom/google/gwt/query/client/Function;)' is deprecated
               [WARN] Line 611: Referencing method 'com.google.gwt.query.client.GQuery.die': method 'com.google.gwt.query.client.GQuery.die()' is deprecated
               [WARN] Line 620: Referencing method 'com.google.gwt.query.client.GQuery.die': method 'com.google.gwt.query.client.GQuery.die(I)' is deprecated
               [WARN] Line 1178: Referencing method 'com.google.gwt.query.client.GQuery.live': method 'com.google.gwt.query.client.GQuery.live(I[Lcom/google/gwt/query/client/Function;)' is deprecated
               [WARN] Line 1187: Referencing method 'com.google.gwt.query.client.GQuery.live': method 'com.google.gwt.query.client.GQuery.live(ILjava/lang/Object;[Lcom/google/gwt/query/client/Function;)' is deprecated
               [WARN] Line 1196: Referencing method 'com.google.gwt.query.client.GQuery.live': method 'com.google.gwt.query.client.GQuery.live(Ljava/lang/String;[Lcom/google/gwt/query/client/Function;)' is deprecated
               [WARN] Line 1205: Referencing method 'com.google.gwt.query.client.GQuery.live': method 'com.google.gwt.query.client.GQuery.live(Ljava/lang/String;Ljava/lang/Object;[Lcom/google/gwt/query/client/Function;)' is deprecated
               [WARN] Line 2411: Referencing method 'com.google.gwt.query.client.GQuery.undelegate': method 'com.google.gwt.query.client.GQuery.undelegate()' is deprecated
               [WARN] Line 2420: Referencing method 'com.google.gwt.query.client.GQuery.undelegate': method 'com.google.gwt.query.client.GQuery.undelegate(Ljava/lang/String;)' is deprecated
               [WARN] Line 2429: Referencing method 'com.google.gwt.query.client.GQuery.undelegate': method 'com.google.gwt.query.client.GQuery.undelegate(Ljava/lang/String;I)' is deprecated
               [WARN] Line 2438: Referencing method 'com.google.gwt.query.client.GQuery.undelegate': method 'com.google.gwt.query.client.GQuery.undelegate(Ljava/lang/String;Ljava/lang/String;)' is deprecated
               See snapshot: /var/folders/xh/1xkfq26532j97q23qw5pdhs40000gn/T/com.google.gwt.query.client.LazyGQueryImpl5264392878806541578.java

```

**Comment by olafleur** [comment]
Could you try with Guava 20 SNAPSHOT and GWTQuery 1.5 beta 1 ?

**Comment by confile** [comment]
Solved with gwtquery-1.4.4-SNAPSHOT
